indico/modules/events/persons/templates/management/pending_persons_list.html

```diff
@@ -0,0 +1,75 @@
+{% extends 'events/persons/management/_persons_list_base.html' %}
+
+{% block title %}
+    {% trans %}Pending persons{% endtrans %}
+{% endblock %}
+
+{% block persons %}
+    <table class="i-table-widget tablesorter">
+        <thead>
+            <tr class="i-table">
+                <th class="i-table checkbox-column" data-sorter="false"></th>
+                <th class="i-table">{% trans %}Name{% endtrans %}</th>
+                <th class="i-table">{% trans %}Email{% endtrans %}</th>
+                <th class="i-table">{% trans %}Affiliation{% endtrans %}</th>
+                <th class="i-table">{% trans %}Event chairperson{% endtrans %}</th>
+                <th class="i-table">{% trans %}Sessions{% endtrans %}</th>
+                <th class="i-table">{% trans %}Contributions{% endtrans %}</th>
+                <th class="i-table">{% trans %}Subcontributions{% endtrans %}</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for person, data in pending_persons.iteritems() %}
+                <tr>
+                    <td class="i-table">
+                        <input type="checkbox" value="{{ person.id }}" class="select-row" name="person_id"
+                                {{ 'disabled' if not person.email }}>
+                    </td>
+                    <td class="i-table">{{ person.full_name }}</td>
+                    <td class="i-contributiontable email-column">{{ person.email }}</td>
+                    <td class="i-table">{{ person.affiliation }}</td>
+                    <td class="i-table">
+                        {{ _('Yes') if data.event_chairperson else _('No') }}
```

Comment:
Oh, I thought we were restricted to just using `{% trans %}` in the html templates!